app.js

```diff
@@ -29,63 +29,80 @@ var storeHoursTotal = [];
 
 var stores = [firstAndPike, seaTacAirport, seattleCenter, capitolHill, alki];
 
+//create table element HEADER FUNCTION BEGIN
```

Comment:
You should break this functionality out into a function! That way you'll be able to use it wherever you want. Your comment implies that this is a function already, but it isn't until you wrap it in a function declaration.